mod-orgs/schemas/organization.json

```diff
@@ -0,0 +1,398 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "The record of an organization",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "The unique UUID for this organization",
+      "type": ["string", "null"],
+      "description": "UUID",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+    },
+    "name": {
+      "description": "The name of this organization",
+      "type": "string"
+    },
+    "code": {
+      "description": "The code for this organization",
+      "type": "string"
+    },
+    "description": {
+      "description": "The description for this organization",
+      "type": "string"
+    },
+    "vendor_status": {
```

Comment:
I know this isn't in the story but please make this "status".  I'll make necessary updates in the various JIRA stories